cosmoz-templatizing-mixin.js

```diff
@@ -234,7 +237,9 @@ export const templatizing = dedupingMixin(base => class extends base { // eslint
 		if (!instance) {
 			return;
 		}
-		const {children, root} = instance;
+		const {
+			children, root
```

Comment:
more of an general question. Should we always make stuff like this on two rows or is this ok?

Reply:
If it were up to me, I'd just throw standard prettier at the whole codebase and be done with it...

Reply:
and keep eslint only for semantic rules (undefined vars, unused vars, etc)

Reply:
Well it's a tricky rule. For objects, the rule now is that properties should be written on their own lines if there are two or more, but the same enforcement doesn't work for destructuring and imports even though they look similar. But I think it makes sense to write out object syntax over multiple lines, the diffs get cleaner and from most aspects it should be easier to read as well.